src/main/webapp/database-tables.js

```diff
@@ -0,0 +1,75 @@
+function showDatabase() {
+  const search = window.location.search;
+  const queryString = '/data-from-db' + search;
+  console.log(queryString)
```

Comment:
Maybe the query string is something we don't need to log because we can see the URL?

Reply:
I think I did that because at this point the page we're on is main-page.html. So I get the part after the ? and concat it to the data-from-db